TestData/Bad/BadUnitTests/DanglingFile/BadUnitTests.swift

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
Why was this file even added?  It does not do anything...

Reply:
it is a part of the test, this is not included in the target